.github/workflows/pull-request.yml

```
@@ -462,3 +462,9 @@ jobs:
         run: |
           echo "Error: Pull request target branch must be 'develop-postgres'. Please refer PR_GUIDELINES.md"
           exit 1
+
+  Validate-Coderabbit:
+    name: Validate CodeRabbit Approval
+    uses: ./.github/workflows/scripts/validate-coderabbit.yml
```

actionlint (1.7.4)

error while parsing reusable workflow "./.github/workflows/scripts/validate-coderabbit.yml": yaml: line 17: mapping values are not allowed in this context

(workflow-call)